Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Booking.hs

```diff
@@ -171,3 +176,29 @@ cancelBookings bookingIds now = do
         BookingUpdatedAt =. val now
       ]
     where_ $ tbl ^. BookingTId `in_` valList (toKey <$> bookingIds)
+
+findByDriverIdTripEndLatLonIfRideStatusNew :: (Transactionable m, MonadTime m) => Id Person.Person -> m (Maybe MT.LatLong)
+findByDriverIdTripEndLatLonIfRideStatusNew driverId = do
```

Comment:
the name seems to be misleading according to what it's doing, can you change the name ?